<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <widget class="QLineEdit" name="expressiontext">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="digitinput">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <widget class="QPushButton" name="digitbutton01">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton00">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton04">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton07">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton02">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="operatorbuttonsqrt">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
      <widget class="QPushButton" name="operatorbuttonadd">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton03">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton05">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pointbutton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton09">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton08">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QPushButton" name="digitbutton06">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backspacebutton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;--</string>
       </property>
      </widget>
      <widget class="QPushButton" name="operatorbuttondivision">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>130</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
      <widget class="QPushButton" name="operatorbuttonmultiple">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
      <widget class="QPushButton" name="operatorbuttonminus">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="operatorbuttonreminder">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
      <widget class="QPushButton" name="operatorbuttonequal">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>51</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
